strawberry_django/mutations/fields.py

```diff
@@ -331,9 +336,11 @@ def resolver(
         data: Any = kwargs.get(self.argument_name)
         vdata = vars(data).copy() if data is not None else {}
 
-        pk = get_pk(vdata)
+        pk = get_pk(vdata, self.key_attr)
```

Comment:
This will be needed if accepting my suggestion of adding a `*` to force this as a kw argument

```suggestion
        pk = get_pk(vdata, key_attr=self.key_attr)
```